spec/support/common_actions.rb

```diff
@@ -1,10 +1,14 @@
 Dir["./spec/support/common_actions/*.rb"].each { |f| require f }
+Dir["./spec/support/custom/**/**.rb"].each { |f| require f }
```

Comment:
It looks a bit strange to have only `support/common_actions` for CONSUL support files but everything under `support/custom` (whether common actions or not) under custom files :thinking:. We're inside the `CommonActions` module, after all.

Two possibilities come to mind. The first one:

```suggestion
Dir["./spec/support/common_actions/custom/*.rb"].each { |f| require f }
```

The second one:

```suggestion
Dir["./spec/support/custom/common_actions/*.rb"].each { |f| require f }
```

Not sure which one I prefer (assuming they even work :smile:).

Reply:
Adding that line to the `spec_helper.rb` file is fine with me :ok_hand:.

Reply:
Oh, sorry, my bad :pray:.

The files under `spec/support/custom/**/*.rb` are already included in `spec/support/**/*.rb`, so the new line wouldn't do anything :thinking:.

I think we'd still need to change the `common_actions.rb` file in order to ensure all the custom common actions files are loaded _before_ the content of the `common_actions.rb` file.

Reply:
Apologies not accepted! I should have tested it before suggesting it! 🙏🏼

Reply:
I'm reading the last paragraph of my previous message and I'm not sure I understand it :rofl:. In summary, my initial suggestions regarding this line still apply.